ui/public/locales/en.json

```
@@ -1272,8 +1283,11 @@
 "label.lang.spanish": "Spanish",
 "label.last.updated": "Last Update",
 "label.lastannotated": "Last annotation date",
+"label.lastboottime": "Boot time of the management server machine",
 "label.lastname": "Last Name",
 "label.lastname.lower": "lastname",
+"label.lastserverstart": "Last start time for this management server",
```

May be, not required to explicitly mention _'this management server_' if this is shown as specific management server details. Update other relevant labels if applicable.

```suggestion
"label.lastserverstart": "Last start time",
```

---

how about `Last Management Server Process start time`? We discussed before that it must be clear whether it is the machine or the java process.